src/constants.ts

```diff
@@ -23,10 +34,21 @@ export const revisions = {
     invalid: invalidRevisions,
 }
 
+export const transferDetails: Record<'test' | 'main', TransferDetails> = {
+    test: {
+        firstBlock: 18878432,
+        lastBlock: 18878436,
+        transferCount: 50 * 5,
+    },
+    main: {
+        firstBlock: 18829247,
+        lastBlock: 18829251,
+        transferCount: 50 * 5,
+    },
+}
+
 const faucetMnemonic =
-    'denial kitchen pet squirrel other broom bar gas better priority spoil cross'.split(
-        ' ',
-    )
+    'denial kitchen pet squirrel other broom bar gas better priority spoil cross'
```

Comment:
Is this mnemonic still needed?

Reply:
This mnemonic was used as a default value, when no mnemonic or private keys were provided, it can be removed in favor of making either of these values mandatory

Reply:
https://github.com/vechain/thor-e2e-tests/blob/5208fed724c36bd87fbe8e0b65e27eac34d1d245/src/test-env.ts#L71